### PR TITLE
Flesh out intro more

### PR DESCRIPTION
**Pull request by adurbin-rivos**
- Add expectations of releases
- Discuss unique identifers of requirements
- Mention test suite

### CODE REVIEW

File: intro.adoc [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+The Boot and Runtime Services (BRS) specification provides the requirements for system vendors and Operating System Vendors (OSVs) to interoperate with one another by providing expectations for the Operating System (OS) to utilize in acts of device discovery, system management, and other rich operations provided in this specification. The approach taken is to leverage industry standards as well as RISC-V ratified specifications.
 
-There are two solutions initially focused on in the form of recipes that describe the various requirements needed for each. The recipe names are named BRS-I and BRS-B, interoperable and bespoke, repsectively.
+The BRS specification is targeting systems that implement HS/S/U privilege modes. This is the expected deployment for OSVs and system vendors in a typical ecosystem covering client systems up through server systems where software is provided by different vendors than the system vendor.
+
+=== Releases
+
+It is expected that the Boot and Runtime Services specification will periodically release a new specification. The determiniation of a new release will be based on the evaluation of significant changes to its underly dependencies.
```
Comment (paul-walmsley-sifive) [inline intro.adoc:10]: s/determiniation/determination/

File: intro.adoc [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+The Boot and Runtime Services (BRS) specification provides the requirements for system vendors and Operating System Vendors (OSVs) to interoperate with one another by providing expectations for the Operating System (OS) to utilize in acts of device discovery, system management, and other rich operations provided in this specification. The approach taken is to leverage industry standards as well as RISC-V ratified specifications.
 
-There are two solutions initially focused on in the form of recipes that describe the various requirements needed for each. The recipe names are named BRS-I and BRS-B, interoperable and bespoke, repsectively.
+The BRS specification is targeting systems that implement HS/S/U privilege modes. This is the expected deployment for OSVs and system vendors in a typical ecosystem covering client systems up through server systems where software is provided by different vendors than the system vendor.
```
Comment (paul-walmsley-sifive) [inline intro.adoc:6]: Consider mentioning that HS mode is optional.  Something like

"The BRS specification is targeting systems that implement S/U privilege modes, and optionally the HS privilege mode."

File: intro.adoc [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+
+=== Releases
+
+It is expected that the Boot and Runtime Services specification will periodically release a new specification. The determiniation of a new release will be based on the evaluation of significant changes to its underly dependencies.
+
+=== Approach to Solutions
+
+The Boot and Runtime Services specification focuses on two solutions in the form of what is deemed a recipe. Each recipe contains the requirements needed to fulfill each solution. The requirements of each recipe will be marked accordingly with an unique identifer. The recipe names are named BRS-I and BRS-B, interoperable and bespoke, repsectively.
```
Comment (paul-walmsley-sifive) [inline intro.adoc:14]: Consider capitalizing Interoperable and Bespoke

File: intro.adoc (hunk 2 above)
Comment (adurbin-rivos) [reply intro.adoc:6]: sg. I'll adjust.

File: intro.adoc (hunk 1 above)
Comment (adurbin-rivos) [reply intro.adoc:10]: will do

File: intro.adoc (hunk 3 above)
Comment (adurbin-rivos) [reply intro.adoc:14]: makes sense.

**Pull request merged**